src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -340,50 +439,49 @@ export default class PhotoField extends React.Component {
                 <span className="cropper-control-label">Make larger<i className="fa fa-search-plus"></i></span>
               </button>}
             </div>
             <div className="cropper-control-container">
               <div className="cropper-control-row">
-                {smallScreen && <button className="cropper-control cropper-control-label-container va-button va-button-link" type="button" onClick={this.zoomOut}>
+                {smallScreen && <button className="cropper-control cropper-control-label-container va-button va-button-link" type="button" onClick={() => this.handleZoom('OUT')}>
                   <span className="cropper-control-label">Make smaller</span>
                 </button>}
-                {smallScreen && <button className="cropper-control cropper-control-label-container va-button va-button-link" type="button" onClick={this.zoomIn}>
+                {smallScreen && <button className="cropper-control cropper-control-label-container va-button va-button-link" type="button" onClick={() => this.handleZoom('IN')}>
                   <span className="cropper-control-label">Make larger</span>
                 </button>}
               </div>
               {[
                 [{
-                  labelText: 'Move up',
-                  className: 'fa fa-arrow-up',
-                  onClick: this.moveUp
+                  direction: 'up',
+                  disabled: this.state.moveUpDisabled
                 }, {
-                  labelText: 'Move down',
-                  className: 'fa fa-arrow-down',
-                  onClick: this.moveDown
+                  direction: 'down',
+                  disabled: this.state.moveDownDisabled
                 }],
                 [{
-                  labelText: 'Move left',
-                  className: 'fa fa-arrow-left',
-                  onClick: this.moveLeft
+                  direction: 'left',
+                  disabled: this.state.moveLeftDisabled
                 }, {
-                  labelText: 'Move right',
-                  className: 'fa fa-arrow-right',
-                  onClick: this.moveright
+                  direction: 'right',
+                  disabled: this.state.moveRightDisabled
                 }]
               ].map((row, index) => (
                 <div className="cropper-control-row" key={index}>
                   {row.map((button) => (
-                    <button className={moveControlClass} type="button" onClick={button.onClick} key={button.className}>
-                      <span className="cropper-control-label">{button.labelText}<i className={button.className}></i></span>
+                    <button className={`${moveControlClass} ${classNames({ disabled: button.disabled })}`} type="button" onClick={() => this.handleMove(button.direction)} key={button.direction}>
```

Comment:
You could consolidate the class name prop into `classNames(moveControlClass, { disabled: button.disabled })`